pallets/xcmp-queue/src/lib.rs

```diff
@@ -263,21 +263,26 @@ pub mod pallet {
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config> {
 		/// Some XCM was executed ok.
-		Success(Option<T::Hash>),
+		Success { message_id: Option<T::Hash>, weight: Weight },
```

Comment:
I would still opt to call it the message hash instead of ID, because the hash isn't necessarily unique when two XCMs containing the exact same instructions will have the same hash, but not necessarily the same ID.

Reply:
sounds good, let me change that